configs/azure.json

```diff
@@ -2,17 +2,18 @@
     "provider": "Azure",
     "description": "Azure estimates based on April 2019 advertised prices",
     "CPU": "0.03900",
-    "spotCPU": "0.007764", 
-    "RAM": "0.001917", 
+    "spotCPU": "0.007764",
+    "RAM": "0.001917",
+    "GPU": "0.0428925",
```

Comment:
What was the source of the price?

Reply:
I did some math to figure out the price of a GPU for 3 different instance types in Azure then I averaged them. It seemed to make sense but probably best to ask an engineer to verify.

https://docs.google.com/spreadsheets/d/18Ugh-Di9zXp3mduOWpTAhiRx7kl-Lzh4wRQ3TFtCl1c/edit?usp=sharing